pkg/signatures/cosign_sig_verifier.go

```diff
@@ -16,8 +16,10 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sigstore/cosign/v2/cmd/cosign/cli/fulcio"
```

Comment:
```suggestion
	"github.com/sigstore/fulcio"
```

Reply:
The `cli/fulcio` is the only place I could find for the `GetRoots` function, so let's keep it.